src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/desktop/ReplyCreatorDesktop.java

```diff
@@ -1,7 +1,9 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.desktop;
 
+import com.wikia.webdriver.elements.mercury.components.discussions.mobile.ReplyCreatorMobile;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.elements.mercury.components.discussions.mobile.ReplyCreatorMobile'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)